internal/cli/atlas/search/index_opts.go

```diff
@@ -77,7 +78,7 @@ func (opts *IndexOpts) NewSearchIndex() (*atlasv2.ClusterSearchIndex, error) {
 	if len(opts.Filename) > 0 {
 		index := &atlasv2.ClusterSearchIndex{}
 		if err := file.Load(opts.Fs, opts.Filename, index); err != nil {
-			return nil, err
+			return nil, fmt.Errorf(failedToLoadIndexMessage, err.Error())
```

Comment:
note: I've investigated adding the customized error to file.Load so that this could go into all commands, but figured this could be a breaking change if any customer is actively parsing output for unmarshal errors 🙁